src/library/librarycontrol.cpp

```diff
@@ -389,14 +388,18 @@ void LibraryControl::slotMoveFocus(double v) {
 }
 
 void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
-    // Ensure a valid widget has the keyboard focus
-    auto focusWidget = QApplication::focusWidget();
-    if (!focusWidget) {
+    // Ensure a valid library widget has the keyboard focus.
+    // QApplication::focusWidget() is not sufficient here because it
+    // would return any focussed widget like WOverview, WWaveform, QSpinBox
+    VERIFY_OR_DEBUG_ASSERT(m_pSidebarWidget && m_pLibraryWidget) {
```

Comment:
like this?

Reply:
Don't use && in assert statements. Otherwise you will not know which operand caused the violation. Please use separate checks as I suggested.

Reply:
alright. I read it as 'separate checks in different places' like for m_pSidebarwidget above..